analyzers/tests/SonarAnalyzer.UnitTest/Rules/ParameterValidationInAsyncShouldBeWrappedTest.cs

```diff
@@ -33,6 +33,13 @@ public class ParameterValidationInAsyncShouldBeWrappedTest
 
 #if NET
 
+        [TestMethod]
+        public void ParameterValidationInAsyncShouldBeWrapped_CSharp10() =>
+            builder
+                .AddPaths("ParameterValidationInAsyncShouldBeWrapped.CSharp10.cs")
```

Comment:
Formatting: Always use the same line-break pattern within the file.
```suggestion
            builder.AddPaths("ParameterValidationInAsyncShouldBeWrapped.CSharp10.cs")
```